docs/data_management/lts/tutorial/lts_tutorial.md

```diff
@@ -0,0 +1,119 @@
+---
+toc_depth: 3
+---
+# Basic Workflow with LST and s3cmd
+
+This tutorial covers setting up and using S3 with LTS, including installing s3cmd, configuring your environment, creating and managing S3 buckets, and setting access permissions on Cheaha.
```

Comment:
We'll need to add more introductory material about why people might want LTS and s3cmd.

Also introduce them to the plan for the tutorial. "In this tutorial, we will show you how to ...". Then the tutorial should be explainable in a single sentence that follows the entire thread. But stick to a single thread. The tutorial should be focused on accomplishing one common goal. We can build on that thread later.

In this case, let's build a tutorial for "I am a researcher who wants to move a file from Cheaha to my individual LTS account for my own personal backup."

Later we can add use-cases to the tutorial, in separate sections, that change that sentence in helpful ways. Individual -> Shared, or File -> Folder. Then we can write about what that would take, using the original tutorial as a guide.

Reply:
The  tutorial is revised: Focuses on individual LTS accounts, then shared LTS account content will be covered in a separate tutorial.